### DISCUSSION TITLE
예약 신청 페이지에 들어가는 2가지 방법이 다 똑 같은 건가요?

### DISCUSSION THREAD

**Discussion by gitjung** [changdoc/naver-vaccine-macro#170] [Q&A]
개발자님,안녕하세요. 
제가 오늘 최신 버전인 1.1.1를 다시 설치해서 진행했는데요.

#4 에서 말씀하신 각각 orgCd와 sid 자리에 복사하여 
이링크 https://v-search.nid.naver.com/reservation/standby?orgCd=11100109&sid=11814857 에 넣어서 
예약 신청 페이지로 이동하는 방법 과 
매그로 확장프로그램의 제일 하단에 나오는 예약신청 목록중 병원이름 되에 있는 열기 버튼으로 클릭해서 예약신청 페이지에 들어가는 방법의 차의 점이 뭐예요?

두 가지 방법중에서 아무거나 사용하면 되나요?
아니면 제가 이해가 잘 못 된건가요?

**Comment by changdoc** [comment]
둘 다 동일합니다.
최초 버전에서는 목록 생성 기능이 없었는데 필요에 의해 기능이 추가됐다고 보시면 되겠네요.

**Reply by gitjung** [reply]
감사합니다!
